api/v1/logcollector_types.go

```diff
@@ -45,6 +45,9 @@ const (
 	CollectProcessPathDisable CollectProcessPathOption = "Disabled"
 )
 
+// EncryptionOption specifies traffic encryption mode to the Syslog server.
```

Comment:
// EncryptionOption specifies the traffic encryption mode when connecting to a Syslog server

Reply:
Updated